### DISCUSSION TITLE
Select special object in multiple object scene

### DISCUSSION THREAD

**Discussion by r0zbeh** [google/model-viewer#4956] [Q&A]
how can select special object in scene that contain multiple ? and how can do something like increase scale when click specific object?

**Comment by phovorka** [comment]
try this [https://modelviewer.dev/examples/scenegraph/#pickMaterialExample](url)

**Reply by r0zbeh** [reply]
any way pick by name?

**Reply by phovorka** [reply]
i think it returns name of the material.... rest functionality U can make by youself...
Scale: [https://modelviewer.dev/examples/scenegraph/#transforms](https://modelviewer.dev/examples/scenegraph/#transforms )
